test/scenarios/bookshop/read.test.js

```diff
@@ -42,6 +42,7 @@ describe('Bookshop - Read', () => {
   test('Books with groupby with path expression and expand result', async () => {
     const res = await GET(
       '/admin/Books?$apply=filter(title%20ne%20%27bar%27)/groupby((author/name),aggregate(price with sum as totalAmount))',
+      //> SELECT author { name }, sum(price) as totalAmount from Books where title != 'bar' GROUP BY author.name
```

Comment:
or, you just need to ensure that you select the author's name in SQL

```suggestion
      //> SELECT author.name, sum(price) as totalAmount from Books where title != 'bar' GROUP BY author.name
```